app/views/content/early-years-teaching-training.md

```diff
@@ -96,12 +80,34 @@ Bursaries are available, including:
 * £4,000 if you have a 2:1 degree
 * £2,000 if you have a 2:2 degree
 
+You are not eligible for a bursary if you have a third class degree.
+
+You do not need to do anything to access this funding. You can [learn more about early years initial teacher training funding](https://www.gov.uk/guidance/early-years-initial-teacher-training-2023-to-2024-funding-guidance).
+
+### Postgraduate employment-based training
+This is a route for graduates working in an early years setting who want to achieve early years teacher status.    
+
 $EY-graduate-employment$
 
-This one-year part-time route is for graduates working in an early years setting who need further training to meet the [early years teachers' standards](https://www.gov.uk/government/publications/early-years-teachers-standards).
+Up to £14,000 of funding is available. This would cover course fees of up to £7,000, plus £7,000 for your employer, to help cover their extra costs while you train. 
```

Comment:
Does this sound slightly odd? The funding isn't available to candidates (if I'm reading it right) so it isn't it just about highlighting the additional (up to) £7k for your employer?

(NB I appreciate that it says this in prod at the moment!)